README.md

```diff
@@ -29,3 +29,487 @@ Transaction submitted through the relay. View transaction on etherscan https://g
 Transaction mined in block 17036120
 Done
 ```
+
+### List of public rpc & relayers for withdrawal
+
```

Comment:
RPC providers fetched from https://ethereumnodes.com/ and https://github.com/ethereum-lists/chains/blob/master/_data/chains/eip155-1.json